internal/provider/backgroundworker/resource/internal/create.go

```diff
@@ -23,9 +25,9 @@ func CreateServiceRequestFromModel(ownerID string, plan backgroundWorker.Backgro
 
 	servicePlan := client.PaidPlan(plan.Plan.ValueString())
 
-	pullRequestPreviewsEnabled := client.BackgroundWorkerDetailsPOSTPullRequestPreviewsEnabledNo
+	pullRequestPreviewsEnabled := client.PullRequestPreviewsEnabledNo
```

Comment:
Hmmm might be some convention I'm missing. Is this being declared in one of the autogenerated code? 

I'm trying to figure out if this is a constant and I should be reading this similar to: 
``` 
prPreviewsEnabled := false 
if plan.PRPreviewEnabled {
  prPreviewsEnabled = true
}
```

which makes the most sense to me. But without being able to find where this was generated, I'm not 100% sure.

Reply:
Yes, client.PullRequestPreviewsEnabledNo is defined in generated file types_gen.go and is a string, not a boolean.